Tools/TestWebKitAPI/Tests/IPC/ConnectionTests.cpp

```diff
@@ -55,6 +55,14 @@ class MockConnectionClient final : public IPC::Connection::Client {
         ASSERT(m_messages.isEmpty()); // Received unexpected messages.
     }
 
+    Vector<MessageInfo> takeMessages()
+    {
+        Vector<MessageInfo> result;
```

Comment:
This is testing code so I don't care much but it seems we could avoid memory allocations like so:
```
auto result = std::exchange(m_messages, Vector<MessageInfo>());
result.reverse();
return result;
```

Reply:
Used deque